### ISSUE TITLE
Parallel landings and takeoffs

### ISSUE THREAD

**Issue by kaitsu71** [zlsa/atc#699]
Would parallel landings and takeoffs be possible to implement without separation penalty?
http://atcradarsim.com/parallel-landings-takeoffs-at-sfo/

**Comment by erikquinn** [comment]
For sure! The only thing that's tough is how we deal with differentiating. For example:
- some facilities can't do any simultaneous parallel stuff
- some facilities can do simultaneous parallel _dependent_ approaches
- some facilities, when they have a "final monitor" position staffed, can do simultaneous parallel _independent_ approaches

So it's hard to achieve all of the below at the same time:
- Busy airports with the appropriate authorizations and runway configurations (like SFO, ORD, IAD, ATL) should be able to run aircraft on final with no required lateral separation (except to remain clear of the NTZ)
- Tiny airports that aren't allowed to do any simultaneous operations due to runway configurations should require 3 mile separation regardless
  - Airports that (a) are like SFO/ORD/IAD/ATL but without "final monitor" staffed, or (b) not authorized for simultaneous independent operations (like KPIT) need to have the appropriate dependent parallel approach minimums applied

To solve it, we could add yet another property to the airport files I guess, specifying whether or not they are authorized for simultaneous independent parallel approaches, simultaneous dependent parallel approaches, or neither. I just didn't think we were really there yet in terms of dealing in those sorts of details personally.

I will admit, it would be nice to have when you're working at some of these airports with triple parallels... running staggers all day long is a wonderful challenge, but not entirely real-world accurate...

**Comment by erikquinn** [comment]
Anybody else want to weigh in on this?

**Comment by n8rzz** [comment]
TL;DR totally possible, but it will take some work.

> To solve it, we could add yet another property to the airport files I guess, specifying whether or not they are authorized for simultaneous independent parallel approaches, simultaneous dependent parallel approaches, or neither. I just didn't think we were really there yet in terms of dealing in those sorts of details personally.

I think you'd have a hard time doing this without adding a new property.  But, once it's there, I'd say that it would be very possible to handle those different rulesets.  This would require: 
- some reworking of the `Runway` model 
- the possible addition of a `RunwayCollection` class that would allow for things like supplying active runway based on wind (I know there is a ticket for this one somewhere), and determining which runways are parallel, and so on.

**Comment by erikquinn** [comment]
The ATC repository is being migrated to it's new home at [https://github.com/openscope/openscope](https://github.com/openscope/openscope), 
and thus, all issues are being closed. If this is still an issue with the latest version of the sim 
(accessible at [http://www.openscope.co](http://www.openscope.co)), or is a feature you still think we are lacking, 
please reopen the issue at the new repo.

*Please note that the vast majority of these issues have been copied to the new repository, or else are covered by other issues created there. See the below screenshot for what it looks like when your issue is known in the new repo:*

![image](https://cloud.githubusercontent.com/assets/5103735/22630816/5a32d794-ebce-11e6-9c46-d61a262e2b23.png)


 Thank you!


Closing this issue.
